Underanalyzer/Mock/GameContextMock.cs

```diff
@@ -28,24 +55,31 @@ public void DefineMockAsset(int assetIndex, string assetName, AssetType assetTyp
         }
         assets[assetIndex] = assetName;
     }
-    public string GetMockAsset(int assetIndex, AssetType assetType)
+    
+    /// <summary>
+    /// Fetches an asset from <see cref="MockAssets"/>.
+    /// </summary>
+    /// <param name="assetType">The asset type of the asset that should be fetched.</param>
+    /// <param name="assetIndex">The index of the asset that should be fetched.</param>
+    /// <returns></returns>
+    public string GetMockAsset(AssetType assetType, int assetIndex)
     {
-        if (MockAssets.TryGetValue(assetType, out var dict))
+        if (!MockAssets.TryGetValue(assetType, out var dict)) return null;
+
+        if (dict.TryGetValue(assetIndex, out string name))
         {
-            if (dict.TryGetValue(assetIndex, out string name))
-            {
-                return name;
-            }
+            return name;
         }
         return null;
     }
-
+    
+    /// <inheritdoc/>
     public string GetAssetName(int assetIndex, AssetType assetType)
     {
         return assetType switch
         {
             AssetType.RoomInstance => assetIndex >= 100000 ? $"inst_id_{assetIndex}" : null,
-            _ => GetMockAsset(assetIndex, assetType)
+            _ => GetMockAsset(assetType, assetIndex)
```

Comment:
We probably also want to change the `GetAssetName` parameter order as well. This was originally done in the `index, type` order for consistency.

Reply:
missed this too.